src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAdjustmentMapping.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.exception.InvalidMappingException;
+import uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.HEARING_LOOP;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.SIGN_LANGUAGE_INTERPRETER;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.STEP_FREE_WHEELCHAIR_ACCESS;
+
+public final class HearingsAdjustmentMapping {
+
+    private HearingsAdjustmentMapping() {
+
+    }
+
+    public static List<Adjustment> getIndividualsAdjustments(HearingOptions hearingOptions) throws InvalidMappingException {
+        if (isNull(hearingOptions)) {
+            return new ArrayList<>();
+        }
+        return getAdjustments(hearingOptions.getArrangements());
+    }
+
+    public static List<Adjustment> getAdjustments(List<String> ccdAdjustments) throws InvalidMappingException {
```

Comment:
minor/optional: think we can make this method cleaner a bit.. feel free to ignore me if you disagree;

don't create the arraylist at the start
return a Collections.emptyList() if isEmpty()

otherwise use a stream over ccdAdjustments 
-> map to getAdjustment
-> collect to list & return